src/com/owncloud/android/ui/fragment/SearchShareesFragment.java

```diff
@@ -139,6 +141,8 @@ public boolean onQueryTextChange(String newText) {
             }
         });
 
+        view.setMinimumWidth(R.dimen.share_user_groups_width);
```

Comment:
I'm curious, why did you move this to code? Wasn't working fine with the old declaration in ```res/layout/search_users_groups_layout.xml``` ?